public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
- I think it would be awesome if you do not commit the `public` folder. Only commit production code.